test/ranges-test.cc

```diff
@@ -65,6 +67,9 @@ TEST(ranges_test, format_vector) {
   EXPECT_EQ(fmt::format("{:n}", vvc), "['a', 'b', 'c'], ['a', 'b', 'c']");
   EXPECT_EQ(fmt::format("{:n:n}", vvc), "'a', 'b', 'c', 'a', 'b', 'c'");
   EXPECT_EQ(fmt::format("{:n:n:}", vvc), "a, b, c, a, b, c");
+
+  EXPECT_FALSE((fmt::is_formattable<unformattable>::value));
```

Comment:
Are extra parentheses needed here and in other places below?

Reply:
No. They were in the original (I moved this assertion up from the tuple one) so I just went with it, but I can remove the unnecessary ones.